cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/metricdefinition/KafkaMetricDef.java

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.monitor.metricdefinition;
+
+import com.linkedin.cruisecontrol.metricdef.MetricDef;
+import com.linkedin.cruisecontrol.metricdef.MetricInfo;
+import com.linkedin.cruisecontrol.metricdef.ValueComputingStrategy;
+import com.linkedin.kafka.cruisecontrol.common.Resource;
+import com.linkedin.kafka.cruisecontrol.metricsreporter.metric.RawMetricType;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.linkedin.cruisecontrol.metricdef.ValueComputingStrategy.AVG;
+import static com.linkedin.cruisecontrol.metricdef.ValueComputingStrategy.LATEST;
+import static com.linkedin.kafka.cruisecontrol.monitor.metricdefinition.KafkaMetricDef.DefScope.COMMON;
+import static com.linkedin.kafka.cruisecontrol.monitor.metricdefinition.KafkaMetricDef.DefScope.BROKER_ONLY;
+
+
+/**
+ * The metric definitions of Kafka Cruise Control.
+ *
+ * The class maps the raw metric types to metric definitions to be used by the analyzer and linear regression
```

Comment:
No, these metrics are just metrics used by the cluster model. They are not raw metrics because they have already be processed (aggregated, averaged, etc). In the context of linear regression model, some of them will be causal metrics (e.g. bytes in rate) while some others are resultant metrics (e.g. CPU_USAGE). So this depends on the context.